src/_pytest/tmpdir.py

```diff
@@ -48,6 +48,7 @@ def from_config(cls, config) -> "TempPathFactory":
     def mktemp(self, basename: str, numbered: bool = True) -> Path:
         """makes a temporary directory managed by the factory"""
         if not numbered:
+            basename = basename.strip("/")
```

Comment:
this only works correct on unix, please use the `os.path.isabs` and fail

Reply:
Hi, please check new commit